src/schedule.js

```diff
@@ -21,11 +22,11 @@ const Schedule = {
 
 const byDay = (date) => {
   let day = date.getDay();
-  if(Schedule[day]) {
+  if(Schedule[day] || day == '0' || day == '6') {
     return false;
   }
   return {
-    text: `Standup is not scheduled for weekday ${day}.`
+    text: `All engineers must head immediately to the main chamber for an important meeting. [${config('SLACK_MENTION')}]`
```

Comment:
Maybe the best solution would be to make Schedule with `enum` instead of `true/false`? Like `NoMessage / SlackCall / Text`?

Reply:
Added some sort of that. I think it's good enough for what it is this for.

😉